src/main/java/mate/academy/bookstore/dto/user/UserLoginRequestDto.java

```diff
@@ -0,0 +1,19 @@
+package mate.academy.bookstore.dto.user;
+
+import jakarta.validation.constraints.Email;
+import jakarta.validation.constraints.NotBlank;
+import jakarta.validation.constraints.NotNull;
+import jakarta.validation.constraints.Size;
+import lombok.Data;
+
+@Data
+public class UserLoginRequestDto {
+    @Email
+    @NotNull(message = "can't be null")
+    private String email;
+
+    @NotBlank(message = "can't be empty")
+    @NotNull(message = "can't be null")
```

Comment:
NotBlank validates null cases as well https://www.baeldung.com/java-bean-validation-not-null-empty-blank